recipe/bld.bat

```diff
@@ -14,14 +14,21 @@ cmake -G "NMake Makefiles" ^
       -DBUILD_SHARED:BOOL=ON ^
       -DBUILD_TESTS:BOOL=ON ^
       -DBUILD_BENCHMARKS:BOOL=OFF ^
+      -DPREFER_EXTERNAL_SNAPPY:BOOL=ON ^
+      -DPREFER_EXTERNAL_LZ4:BOOL=ON ^
+      -DPREFER_EXTERNAL_ZLIB:BOOL=ON ^
```

Comment:
Since we are not currently including these, might be best to disable them for now. Though agree we should strive to use external packages to meet these needs as much as possible. Already issue ( https://github.com/conda-forge/blosc-feedstock/issues/18 ) is open on this point.